src/lib/y2storage/devices_lists/lvm_lvs_list.rb

```diff
@@ -28,6 +28,33 @@ module DevicesLists
     # List of LVM logical volumes from a devicegraph
     class LvmLvsList < Base
       list_of ::Storage::LvmLv
+
+      # Filesystems located in the logical volumes
+      #
+      # @return [FilesystemsList]
+      def filesystems
+        fs_list = list.map do |lv|
+          begin
+            lv.filesystem
+          rescue ::Storage::WrongNumberOfChildren
+            # No filesystem in the logical volume
+            nil
+          end
+        end
+        FilesystemsList.new(devicegraph, list: fs_list.compact)
+      end
+
+      # Volume groups containing the logical volumes
+      #
+      # @return [LvmVgsList]
+      def lvm_vgs
+        vgs = list.map(&:lvm_vg)
```

Comment:
I would say lvs should always be in a vg. If for some weird reason we have a lv floating around, I would expect `unassigned_lv.lvm_vg`to raise an exception, instead of returning nil.